### DISCUSSION TITLE
How to customize WordPress embeds to match your theme

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#297] [Topic Ideas]
WordPress embeds let any WP site's posts be embed anywhere. 

![image](https://github.com/user-attachments/assets/3efb9595-ba52-49d2-86dd-bb9e1fc8518c)

But did you know that you can create a custom `embed.php` template to customize it? There's no block template equivalent, but there are many hooks that you can filter and even load custom CSS to to make it match your theme.

![image](https://github.com/user-attachments/assets/bf5463f3-dbf6-4473-bf44-2f6c7845971c)

You can even customize the embed code panel:

![image](https://github.com/user-attachments/assets/8bc8547b-af11-426c-b1c7-b46794205777)

This tutorial would explore all the various tips, tricks, and techniques to completely customizing a site's embedded posts so that they match their brand/theme when shared on third-party sites.

A lot of the techniques that would be covered (particularly PHP) are covered in this file:

- https://github.com/x3p0-dev/x3p0-ideas/blob/869d801bd7e11dccffb4b344b174c7b52ac89e52/src/Embeds.php

**Comment by bph** [comment]
This topic was approved in the 9/5 editorial group meeting. Please continue the conversation on the issue
- #306

**Comment by meszarosrob** [comment]
The article is published https://developer.wordpress.org/news/2025/02/customize-wordpress-embeds-to-match-your-theme/ :v:
